x-pack/plugins/apm/server/routes/transactions.ts

```diff
@@ -18,18 +18,18 @@ import { getSpans } from '../lib/transactions/spans/get_spans';
 
 const pre = [{ method: setupRequest, assign: 'setup' }];
 const ROOT = '/api/apm/services/{serviceName}/transactions';
-const defaultErrorHandler = (reply: IReply) => (err: Error) => {
+const defaultErrorHandler = (err: Error) => {
   // tslint:disable-next-line
   console.error(err.stack);
   // @ts-ignore
-  reply(Boom.wrap(err, err.statusCode || 400));
+  Boom.boomify(err, { statusCode: err.statusCode || 400 });
```

Comment:
@joshdover Minor thing: this should be returned or thrown. We've created an issue for it here: https://github.com/elastic/kibana/issues/24844

We'll fix the APM instances (if there are more) but just a heads up if this is also the case for other plugins.